src/object1.cc

```diff
@@ -3013,141 +2971,99 @@ bool_ object_out_desc(object_type *o_ptr, FILE *fff, bool_ trim_down, bool_ wait
 			text_out("It makes you invisible.  ");
 		}
 
-		if (o_ptr->tval != TV_TRAPKIT)
+		vn = 0;
+		if (f2 & (TR2_SUST_STR))
 		{
-			vn = 0;
```

Comment:
Same thing here wrt. indentation. Just leave the surrounding "{" in place.

(I'd like the see the indentation removed eventually, but it can be done after everything is finalized and merged.)